project-security/verifying-signatures.md

```diff
@@ -507,6 +507,19 @@ signed tags or commits on top of them unless you personally vouch for the
 trustworthiness of the unsigned commits. Instead, ask the person who pushed the
 unsigned commits to sign them.
 
+You should always perform this verification on a trusted local machine with
+properly validated keys (which are available in the [Qubes Security
+Pack](/security/pack/)) rather than relying on a third party, such as GitHub.
```

Comment:
_this verification_ has now lost it's referent

Reply:
I changed back to "this verification".